shared_memory_dict/dict.py

```diff
@@ -19,16 +29,29 @@ def cleanup(self) -> None:
         self._memory_block.close()
 
     def move_to_end(self, key: str, last: Optional[bool] = True) -> None:
+        warnings.warn(
+            "The 'move_to_end' method will be removed in future versions. "
+            "Use pop and reassignment instead.",
```

Comment:
Could you change it to a simple quotation?

Reply:
changed!